modules/virt-creating-servicemonitor-resource-for-node-exporter.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events-monitoring/virt-exposing-custom-metrics-for-vms.adoc
+
+[id="virt-creating-servicemonitor-resource-for-node-exporter_{context}"]
+= Creating a ServiceMonitor resource for the node exporter service
+
+To use the metrics exposed by the `node-exporter` service, you must configure {product-title} monitoring to use a Prometheus client library and scrape metrics from the `/metrics` endpoint. You can do this by using a `ServiceMonitor` custom resource definition (CRD) that specifies how the `node-exporter` service should be monitored.
+
+.Prerequisites
+
+* You have access to the cluster as a user with `cluster-admin` privileges or the `monitoring-edit` role.
+* You have enabled monitoring for the user-defined project by configuring the `node-exporter` service.
+
+.Procedure
+. Create a YAML file for the `ServiceMonitor` resource configuration. This configuration defines a `ServiceMonitor` resource that scrapes the metrics exposed by the `node-exporter` service. The following example shows a YAML file with the `ServiceMonitor` that matches any service with the label set to `metrics` and queries the `exmet` port every 30 seconds.
++
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    k8s-app: node-exporter-metrics-monitor
+  name: node-exporter-metrics-monitor <1>
```

Comment:
Add '#' before the '<1>, <2>, <3>...etc.'

Reply:
Please see my comment above. Same response here.